app/assets/javascripts/i18n.js

```diff
@@ -857,12 +854,43 @@
     }
 
     return scope;
-  }
+  };
+  /**
+   * Merge obj1 with obj2
```

Comment:
Maybe put a note to clarify that this is a shallow merge.
Also obj1 or obj2 is not changed